assets/airports/cyow.json

```diff
@@ -0,0 +1,1279 @@
+                "22": []
+            },
+            "draw": [
+                ["ART*", "CYRIL"],
+                ["BUGSY*", "SAVAL", "TAPVO", "CYRIL"],
+                ["CYRIL", "TEXEN", "ADVOS"],
+                ["CYRIL", "CAVUR", "LITRI", "APMUP", "MERKO", "ILOGI", "PETLU"],
+                ["CAVUR", "AMAGI", "NOVUX", "TILOG", "EBNAP"]
+            ]
+        },
+        "LEAMY3": {
+            "icao": "LEAMY3",
+            "name": "Leamy Three",
+            "entryPoints": {
+                "BEMOG": [["BEMOG", "A210-"]]
+            },
+            "body": [],
+            "rwy": {
+                "07": [["ALIDO", "A160-"], ["DUNUM", "A130-"], "GABEV", ["KEMLU", "A50|S210-"], "SENNZ", ["MODON", "A30"], "VINRA", "#251"],
+                "14": [["ALIDO", "A100-"], "DUNUM", ["RAFNO", "A30"]],
+                "25": [["DUNUM", "A160-"], "MERKO", ["REVAL", "A50|S210"], ["XOLTI", "A30"], "IGTAD", "#072"],
+                "32": [["DUNUM", "A160-"], "MERKO", ["ILUBO", "A50|S210"], ["WATTO", "A30"], "TIDUL", "#141"],
+                "22": []
+            },
+            "draw": [
+                ["BEMOG", "ALIDO", "DUNUM", "RAFNO"],
+                ["DUNUM", "GABEV", "KEMLU", "SENNZ", "MODON", "VINRA"],
+                ["DUNUM", "MERKO", "REVAL", "XOLTI", "IGTAD"],
+                ["MERKO", "ILUBO", "WATTO", "TIDUL"]
+            ]
+        },
+        "MEECH1": {
+            "icao": "MEECH1",
+            "name": "Meech One",
+            "entryPoints": {
+                "SMARE": ["SMARE", ["TOXIP", "A210"]]
+            },
+            "body": [],
+            "rwy": {
+                "07": [["ONDOB", "A200-"], "GABEV", ["KEMLU", "A50|S210-"], "SENNZ", ["MODON", "A30"], "VINRA", "#251"],
+                "14": [["ONDOB", "A100-"], ["RAFNO", "A30"]],
+                "25": [["ONDOB", "A200-"], ["GABEV", "A130"], "MERKO", ["REVAL", "A50|S210"], ["XOLTI", "A30"], "IGTAD", "#072"],
+                "32": [["ONDOB", "A200-"], ["GABEV", "A130"], "MERKO", ["ILUBO", "A50|S210"], ["WATTO", "A30"], "TIDUL", "#141"],
+                "22": []
+            },
+            "draw": [
+                ["SMARE", "TOXIP", "ONDOB", "RAFNO"],
+                ["ONDOB", "GABEV", "KEMLU", "SENNZ", "MODON", "VINRA"],
+                ["GABEV", "MERKO", "REVAL", "XOLTI", "IGTAD"],
+                ["MERKO", "ILUBO", "WATTO", "TIDUL"]
+            ]
+        },
+        "RIVER2": {
+            "icao": "RIVER2",
+            "name": "River Two",
+            "entryPoints": {
+                "VILRO": ["VILRO"],
+                "CATOG": ["CATOG", "OBTAX"]
+            },
+            "body": [["YUL", "A280-"], "ALSET"],
+            "rwy": {
+                "07": [["THURO", "A160-"], "URLAG", ["ADVOS", "A130-"], "AMAGI", ["NOVUX", "A50|S210-"], ["TILOG", "A30"], "EBNAP", "#247"],
+                "14": [["THURO", "A160-"], ["XOLTI", "A50|S230-"], ["ONBAB", "A50"], ["ILOGI", "A35|S210-"], "PETLU", "#320"],
```

Comment:
XOLTI should be `["XOLTI", "A50+|S230-"]`
ILOGI should be `["ILOGI", "A35+|S210-"]`